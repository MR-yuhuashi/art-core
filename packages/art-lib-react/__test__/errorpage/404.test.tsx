import React from 'react';
import { shallow } from 'enzyme';
import Page404 from '../../src/components/errorpage/404';
import toJson from 'enzyme-to-json';

test('404 page test', () => {
  const wrapper = shallow(<Page404 />);
  expect(wrapper.find('div').length).toBe(1);
  expect(wrapper.find('.error-page').exists()).toBeTruthy();
  expect(toJson(wrapper)).toMatchSnapshot();
});