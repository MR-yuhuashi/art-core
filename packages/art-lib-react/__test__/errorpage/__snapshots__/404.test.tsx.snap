// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page test 1`] = `
<div
  className="error-page"
>
  (404)页面不存在，请检查地址
</div>
`;
